src/test/scala/com/crealytics/spark/v2/excel/WriteAndReadSuite.scala

```diff
@@ -48,25 +54,19 @@ object WriteAndReadSuite {
 
   val userDefinedSchema_02 = StructType(
     List(
-      StructField("Day", LongType, true),
-      StructField("Month", LongType, true),
-      StructField("Customer ID", StringType, true),
-      StructField("Customer Name", StringType, true),
-      StructField("Standard Package", IntegerType, true),
-      StructField("Extra Option 1", IntegerType, true),
-      StructField("Extra Option 2", IntegerType, true),
-      StructField("Extra Option 3", LongType, true),
-      StructField("Staff", StringType, true)
+      StructField("Id", IntegerType, nullable = true),
```

Comment:
For bug fix, if it is possible, please add a test-case instead of modify the existing one?
If the existing one is incorrect, we may need to add another MR dedicated for that.
In long term, this is just to avoid a regression issue for existing users.

Reply:
That's what I did initially.  Afterwards, I noticed that userDefinedSchema_02 and expectedData_02 weren't used and I decided to cleanup. I can revert that if you would like me to.